tests/publiccloud/prepare_tools.pm

```diff
@@ -47,7 +47,7 @@ sub run {
         assert_script_run("pip3 install -q keyring");
```

Comment:
Although it is not in the scope of this PR, why we use pip? We don't have package or is old?

Reply:
I don't remember why, since it's been like this quite a long time. @cfconrad ?

Reply:
Problem is in usage of python development repositories in obs. Repository devel:/languages:/python:/backports/SLE_15 have python3-PyYAML 5.x but aws-cli requires version  < 4.0.

Reply:
Yeah, I also don't remember exactly. But there where a lot of problems with dependencies. And it was needed by some of the tools like ipa, az, gcloud or aws.
We could try to remove it!